os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c

```diff
@@ -728,13 +733,20 @@ int8_t cmd_wifi_on(WiFi_InterFace_ID_t interface_id)
 {
 	int ret;
 
+	if(rltk_wlan_running(WLAN0_IDX)) {
+		if (wifi_set_mode(RTW_MODE_STA) < 0) {
+			ndbg("\n\rERROR: Wifi Set Mode to STA failed!\n");
+			return RTW_ERROR;
+		}
+	} else {
 	/* Kill init thread after all init tasks done */

```

Comment:
please check the indents.

Reply:
It is good to match the coding rule for whole codes, but we will not handle the coding rule for vendor code, for example board specific or arch specific codes.